invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessRequests/AccessRequestsTab.js

```diff
@@ -178,10 +178,7 @@ export class AccessRequestsTab extends Component {
                             )}
                           </label>
                           <Form.Field>
-                            <RichInputField
-                              fieldPath="accept_conditions_text"
-                              editorConfig={this.editorConfig}
```

Comment:
there is an usused leftover variable `this.editorConfig` left after this change, could you remove it?